packages/babel-preset-env/test/fixtures/corejs2/entry-all/output.mjs

```diff
@@ -1,3 +1,4 @@
+import "core-js/modules/es6.array.concat.js";
```

Comment:
In core-js2, there is not a `/modules/es6.array.concat.js`. Is this a bug? 

https://github.com/zloirock/core-js/tree/v2/modules